### PR TITLE
[MU4] Concert pitch control

### PR DESCRIPTION
**Pull request by RomanPudashkin**
- implemented the concert pitch control
- actualized a view of the zoom control

concert_pitch

### CODE REVIEW

File: mu4/notation/view/concertpitchcontrolmodel.cpp [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+        style()->styleChanged().onNotify(this, [this]() {
+            emit concertPitchEnabledChanged();
+        });
+    });
+}
+
+bool ConcertPitchControlModel::canReceiveAction(const ActionName&) const
+{
+    return true;
```
Comment (Eism) [inline mu4/notation/view/concertpitchcontrolmodel.cpp:54]: the abstract class method returns true. No need to override

Comment (RomanPudashkin) [reply mu4/notation/view/concertpitchcontrolmodel.cpp]: fixed

**Review by Eism — APPROVED** [review]

**Comment by igorkorsukov** [comment]
needs rebase

**Comment by RomanPudashkin** [comment]
@igorkorsukov fixed

**Pull request merged**